src/main/groovy/net/minecraftforge/gradleutils/changelog/ChangelogUtils.groovy

```diff
@@ -41,7 +41,24 @@ class ChangelogUtils {
      * @param endCommitHash The commit hash of the commit to use as the end of the changelog.
      * @return A multiline changelog string.
      */
+    @Deprecated(forRemoval = true, since = "2.3")
```

Comment:
Why deprecate?

Reply:
Because this method is effectively private, and would be pretty difficult to use on its own outside of the buildscript. It's only ever used by the changelog plugin itself, which no longer uses this method.